languages/csharp/exercises/concept/casting/.docs/instructions.md

```diff
@@ -0,0 +1,72 @@
+Our football club (first encountered in (cross-ref-tba)) is soaring in the leagues, and you have been invited to do some more work, this time on the security pass printing system.
+
+The class hierarchy of the backroom staff is as follows
+
+```
+Team Support (interface)
+    |
+    - Chairman
+    |
+    - Manager
+    |
+    - Staff (abstract)
+        |
+        - Physio
+        |
+        - OffensiveCoach
+        |
+        - GoalKeepingCoach
+        |
+        - Security
+            |
+            - SecurityJunior
+            |
+            - SecurityIntern
+            |
+            - PoliceLiaison
+
+
+```
+
+A complete implementation of the hierarchy is provide as part of the source.
+
+All data passed to the security pass maker has been validated and is guaranteed to be non-null.
+
+## 1. Get display name for a member of the support team as long as they are staff members
+
+Please implement the `securityPassMaker.GetDisplayName()` method. It should return the value of the `Title` field instances of all classes derived from `Staff` and, otherwise, "Too Important for a Laminate".
```

Comment:
```suggestion
Please implement the `SecurityPassMaker.GetDisplayName()` method. It should return the value of the `Title` field instances of all classes derived from `Staff` and, otherwise, "Too Important for a Laminate".
```